packages/Sortable/src/components/SortableItem/SortableItem.js

```diff
@@ -10,27 +10,33 @@ import { itemStyles, itemIndexStyles, itemHandleStyles, itemBodyStyles, itemClos
 
 const propTypes = {
   children: PropTypes.node.isRequired,
+  /** Handle element, default is a handle icon */
+  handle: PropTypes.node,
   /** Indicator to identify the number sequence  */
   hasNumbers: PropTypes.bool.isRequired,
   /** Numerical representation of an item */
   index: PropTypes.number.isRequired,
+  /** If drag action should be disabled on an item */
+  isDragDisabled: PropTypes.bool,
   /** Callback when deleting an item */
   onRemove: PropTypes.func,
 };
 
 const defaultProps = {
+  handle: <HandleIcon />,
+  isDragDisabled: false,
   onRemove: null,
 };
 
-const SortableItem = ({ children, index, hasNumbers, onRemove, ...moreProps }) => {
+const SortableItem = ({ children, index, handle, hasNumbers, isDragDisabled, onRemove, ...moreProps }) => {
   const I18n = useI18n();
 
   const handleRemove = () => {
     onRemove(index);
   };
 
   return (
-    <Draggable draggableId={`draggable-${children.props.sortId}`} index={index}>
+    <Draggable draggableId={`draggable-${children.props.sortId}`} index={index} isDragDisabled={isDragDisabled}>
```

Comment:
Can you please add a story to the `Sortable` component that demonstrates a locked Sortable.Item